app/scripts/metamask-controller.js

```diff
@@ -3877,6 +3888,10 @@ export default class MetamaskController extends EventEmitter {
         userStorageController.setIsProfileSyncingEnabled.bind(
           userStorageController,
         ),
+      syncInternalAccountsWithUserStorage:
+        userStorageController.syncInternalAccountsWithUserStorage.bind(
+          userStorageController,
+        ),
```

Comment:
Nice binding! Do we will be using this in the UI? 😄 
I will probably do the same for network syncing.

Discuss - should we have 1 "sync" function fires off all sync features, or have this for granularity?

Lets stick with this for now.